client/jetpack-connect/authorize-form.jsx

```diff
@@ -110,10 +110,12 @@ class JetpackConnectAuthorizeForm extends Component {
 				? <LoggedInForm
 					{ ...this.props }
 					isSSO={ this.isSSO() }
+					isWCS={ this.props.jetpackConnectAuthorize.queryObject.from === 'woocommerce-services' }
```

Comment:
Let's follow the `isSSO()` pattern here and only have the logic for determining if we came from WCS in one place.